roles/kubernetes-apps/ansible/tasks/cleanup_dns.yml

```diff
@@ -1,31 +1,4 @@
 ---
-- name: Kubernetes Apps | Delete old CoreDNS resources
-  kube:
-    name: "coredns"
-    namespace: "kube-system"
-    kubectl: "{{ bin_dir }}/kubectl"
-    resource: "{{ item }}"
-    state: absent
-  with_items:
-    - 'deploy'
-    - 'configmap'
-    - 'svc'
-  tags:
-    - upgrade
-
-- name: Kubernetes Apps | Delete old nodelocalDNS resources
```

Comment:
Why would you remove this?

Reply:
It's ancient legacy stuff to remove. It wasn't in the last 2 major releases.